murren/models.py

```diff
@@ -27,3 +27,16 @@ def save(self, *args, **kwargs):
             output_size = (300, 300)
             img.thumbnail(output_size)
             img.save(self.murren_avatar.path)
+
+
+class EmailToken(models.Model):
+    murren = models.ForeignKey(Murren, on_delete=models.CASCADE)
+    token = models.CharField(max_length=128)
+    time = models.DateTimeField(auto_now_add=True)
```

Comment:
According to the name of the `time` field, there should be time according to the idea, but in the type we have `models.DateTimeField`, it will be more correct to call it `created_at`